vars/acePushConfigToGit.groovy

```diff
@@ -4,6 +4,7 @@ void call(Map opts = [:]) {
 
   target = readYaml file: 'target-data/target.yaml'
   cfg = readYaml file: opts.aceFile ?: 'ace.yaml'
+  gitOpsfolder=$(mktemp -d gitOpsXXXXXX)
```

Comment:
@tomsolem , @ekarlso , @Starefossen , is this the correct syntax in groovy? I think I might have suggested this to @tomsolem while thinking about this as bash.

Reply:
Yep, this is not Groovy. If you need to run a shell script you can use `sh()`.